packages/core/src/babelPlugins/__tests__/__fixtures__/mock-cell-data/output_TodoListCell.mock.js

```diff
@@ -1,12 +1,13 @@
-import { afterQuery } from './'
-export const standard = afterQuery(
-  mockGraphQLQuery('TodoListCell_GetTodos', () => ({
-    todos: [
-      {
-        id: 1,
-        body: 'Cheese',
-        status: '',
-      },
-    ],
-  }))
-)
\ No newline at end of file
+import { afterQuery } from './TodoListCell.js'
+export const standard = () =>
```

Comment:
Aren't you introducing a breaking change here by converting `standard` into a function, rather than an object?

Reply:
Hmm maybe I'm getting something wrong - but isn't it a function when afterQuery isn't present? (That's what it seemed like)

Reply:
Hmm so I looked through the functions `mockGraphQlQuery` and `mockGraphQl`. https://github.com/redwoodjs/redwood/blob/a32b8b4898326bb00b108916d24b02dc626ce042/packages/testing/src/mockRequests.ts#L69.

If we pass in a function to mockGraphQLQuery, it also returns a function. So in the case of `standard`, without afterQuery it also is a function.

Reply:
This is also how the actual code works

```
  afterQuery = (data) => ({ ...data }),
```

That's the default, so if you don't pass `afterQuery`, it will be a function by default

Reply:
Ah, ok. I forgot that in a mock file "standard" is actually meant to be a function, and not an object.

I was looking at the comment on 89 and assumed that this was going to cause all the current mocks to now import `standard()` rather than just an object called standard.

Reply:
Ok, I've investigated this further and this is a breaking change.

@Tobbe We actually run `afterQuery(myData)` here, so we're returning the result of `afterQuery`, since that it what you want in your `Success` handler.

Reply:
The reason why is this a breaking change is we also allowed standard to be an ordinary object, instead of a function that returns an object:

```js
// TestCell.mock.js
// Define your own mock data here:
export const standard = {
  test: {
    id: 42,
  },
}
```

We then import that mock in the stories file, and use it for our `Success` handler.

```js
// TestCell.stories.js
import { standard } from './TestCell.mock'

export const success = () => {
  return Success ? <Success {...standard} /> : null
}
```

HOWEVER if our Cell exports `afterQuery` the `standard` object will now be a function instead of an expected object. 

(Note: If you're trying to reproduce this there appears to be a caching issue somewhere. You must restart Storybook.)

I think the most sane way to fix this is to ONLY allow the standard export to be a function.

Reply:
I've created an issue to resolve this: https://github.com/redwoodjs/redwood/issues/1801

Reply:
👍 yep, I expect most people will use the function form that the cli generates by default! I suppose this a half fix 😂, atleast you can use afterQuery now.